bin/baseset/orig_dos.obm

```diff
@@ -0,0 +1,76 @@
+; $Id$
```

Comment:
Don't think this is relevant any more.

Reply:
I agree, but I'd say it should be removed from all files at once, rather than one at a time